packages/fxa-auth-server/lib/senders/emails/templates/cadReminderSecond/index.stories.ts

```diff
@@ -12,7 +12,7 @@ export default {
 
 const createStory = storyWithProps(
   'cadReminderSecond',
-  'Sent 72 hours after a user clicks "send me a reminder" on the connect another device page.',
+  'Sent 72 hours after a user clicks "send me a reminder" on the connect another device page. It currently cannot be triggered in staging, only in production.',
```

Comment:
Can we add a reference to the ticket for that last sentence?

Also for `subscriptionsPaymentExpired`, I think we should also either reference the ticket since it is happening in prod, or, just remove the sentence. I'm in favor of the latter since the ticket is being worked on.